src/applications/vaos/tests/appointment-list/components/ConfirmedAppointmentDetailsPage.unit.spec.jsx

```diff
@@ -31,7 +31,9 @@ describe('VAOS <AppointmentsPageV2>', () => {
     const appointment = getVAAppointmentMock();
     appointment.attributes = {
       ...appointment.attributes,
-      startDate: moment().format(),
+      startDate: moment()
+        .tz('America/Denver')
```

Comment:
Generally, I think we've not set the timezone on dates in these appointment mocks, since we normally get them as UTC. The timezone setting in the expect statements below are the important changes, from what I remember.

Reply:
Good call.  Removed.